src/components/Router/index.js

```diff
@@ -137,9 +138,10 @@ class CrossDeviceMobileRouter extends Component {
 
   sendClientSuccess = () => {
     this.state.socket.off('custom disconnect', this.onDisconnect)
-    const { faceCapture } = this.props
-    const data = faceCapture ? {faceCapture: {blob: null, ...faceCapture}} : {}
-    this.sendMessage('client success', data)
+    const captures = Object.keys(this.props.captures).reduce((acc, key) =>
+      acc.concat(omit(this.props.captures[key], ["blob", "base64"])),
```

Comment:
could pick be better than `omit`? thinking of the case where we had stuff to captures

Reply:
Maybe, it also might end up being harder to maintain, though.

Blacklisting trouble-fields, vs. whitelisting required fields - classic toss-up, I guess

I would rather that we just go with blacklisting (as that's what the code was already doing), and just revisiting in the future if blacklisting ends up being a pain